sod/index.html

```diff
@@ -0,0 +1,174 @@
+<!-- Global site tag (gtag.js) - Google Analytics - check if people actually use this tool -->
+
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<script async src="https://www.googletagmanager.com/gtag/js?id=G-NMLK7NHP6Z"></script>
+	<script>
+		window.dataLayer = window.dataLayer || [];
+
+		function gtag() {
+			dataLayer.push(arguments);
+		}
+
+		gtag('js', new Date());
+
+		gtag('config', 'G-NMLK7NHP6Z');
+	</script>
+	<script data-ad-client="ca-pub-3448992463946827" async
+			src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+	<script src="../era/colors.js"></script>
```

Comment:
Note that here, the `colors.js`, `threat.js` and `font.js` are the same, so I'm linking back to the era files. That may change in future, but its not worth creating new files to get out of sync unless it's intended.